src/views/single-event.php

```diff
@@ -22,7 +22,7 @@
 
 ?>
 
-<div id="tribe-events-content" class="tribe-events-single">
+<div id="tribe-events-content" class="tribe-events-single tribe-events-view-single"> <!-- TODO: tribe-events-view-single should be added under condition -->
```

Comment:
you can use `tribe_events_views_v2_is_enabled()` to check if v2 is enabled

Reply:
will have to ask Gustavo, I think we're going to use a new constant for this

Reply:
@paulmskim I removed that class from the div and added a body class `single-tribe_events-v2` via my theme's functions.php for now. I'm using that to target stuff in the style files. Gustavo should be able to properly add that class soon